src/main.rs

```diff
@@ -82,6 +86,37 @@ impl HwMon {
         })
     }
 
+    pub fn pwm_min(&mut self) -> u32 {
+        if self.pwm_min.is_none() {
+            self.pwm_min = Some(
+                self.read("pwm1_min")
+                    .unwrap_or_default()
+                    .parse()
+                    .unwrap_or(0),
+            )
+        };
+        self.pwm_min.unwrap_or(0)
```

Comment:
Please change to `unwrap_or_default()`

Reply:
This was copied from `fan_min()`. Shouldn't it be changed there too?

Reply:
Yes, please change also there